docs/metrics.md

```diff
@@ -125,10 +125,10 @@ Improving Kubevirt's Observability is a important topic and we are currently wor
 
 A design proposal and its implementation history can be seen [here](https://docs.google.com/document/d/1bEwrnZZkVsCtz0PSyzlxOdhupL6GTurkUYcz7TXFM1g/edit)
 
- # Other Metrics 
+ # Other Metrics
 ## kubevirt_vmi_vcpu_wait_seconds
 #### HELP kubevirt_vmi_vcpu_wait_seconds vcpu time spent by waiting on I/O
-## leading_virt_controller
-#### HELP leading_virt_controller Indication for an operating virt-controller.
```

Comment:
Are these alerts documented anywhere? Does anyone make use of them and expect their current name? If so, we have a problem about this rename.
(Sorry for knowing very little of alerts myself)

Reply:
This metric is not used internally anywhere AFAIK (if it was, I would have changed that as well).  However, it is exported and a user can query this in prometheus.

Reply:
How would a user know about these metrics? Are you aware of users that already track them? We may consciously decide to break current users, but we should estimate the cost of the breakage vs the benefit of this rename. I think these should be discussed in the PR text.

Reply:
I don't know if there's a way to know who's using those, perhaps @sradco can help out here

Reply:
Hi @dankenigsberg ,

The alerts are not documented yet. 
The existing metrics are documented upstream and they need d/s documentation.

There is an epic for each team to document the alerts and provide steps for "what to do"  when the alert fires,it called 
Standard Operating Procedures (SOPs) for each alert. Think of this as a runbook for each alert, for what to do when it happens under which conditions.

Its part of the requirements to reach the "Deep Insights" level. 
The renaming is for us, to be able to better filter the metrics and alert and to align to OCP's naming conventions.

They have the metrics both in Prometheus and also if we/they create a Grafana dashboard, then they will get all these metrics with the "kubevirt_" prefix in the drop-down.

We don't use any of the metrics at this time in the OCP UI, except for 1 metric. For that one we will handle the UI change and only after it is released we will deprecate the old one.
We dont have a Grafana dashboard at this time, so I do not expect issues for the users.

Reply:
Thanks @sradco. We are at upstream, and this is an upstream-only discussion. I'm fine with "breaking" these undocumented names, particularly if you believe nobody depends on them; I'm just asking to say so in the PR text.